SplitFiles/slot.c

```diff
@@ -52,8 +52,8 @@ void slot_func(GameComponent* input, GameParameters* params) {
       params->messageSourceAddress = (void *)state;
       state->bread = NULL;
       state->power = 0;
-      params->effect = TUNE_EFFECT_UP; 
-      //bit_fx(BFX_EXPLOSION);
+      params->effect = TUNE_EFFECT_UP;
+      printf(PRINTAT"%c%c%c", state->xCoord + 7, state->yCoord - 1, ' ');
```

Comment:
same here

Reply:
Could do this writing one byte to the contended memory using the "White on White" trick I used for the customer list I suppose